apps/modernization-ui/src/permissions/useSystemManagementAccess.ts

```diff
@@ -0,0 +1,21 @@
+import { useMemo } from 'react';
+
+const SYSTEM_MANAGEMENT_PERMISSIONS = [
+    'EPILINKADMIN-SYSTEM',
+    'VIEWELRACTIVITY-OBSERVATIONLABREPORT',
+    'SRTADMIN-SYSTEM',
+    'VIEWPHCRACTIVITY-CASEREPORTING',
+    'IMPORTEXPORTADMIN-SYSTEM',
+    'REPORTADMIN-SYSTEM',
+    'ALERTADMIN-SYSTEM'
+];
+
+const useSystemManagementAccess = (permissions: string[] | undefined | null) => {
```

Comment:
This should be a hook that returns the permissions for the navigation bar, not just for System Management Access.  For now it should return an object with only the boolean property `systemManagementAccess`.